pkg/storage/reports/critical_localities_report.go

```diff
@@ -110,7 +110,7 @@ func (r *replicationCriticalLocalitiesReportSaver) loadPreviousVersion(
 
 		// if the row is nil then this is the first time we are running and the reload is needed.
 		if row != nil {
-			generated, ok := row[0].(*tree.DTimestamp)
+			generated, ok := row[0].(*tree.DTimestampTZ)
 			if !ok {
 				return errors.Errorf("Expected to get time from system.reports_meta but got %+v", row)
```

Comment:
if you change the error above to use `row[0]` probably you want the same here.

Reply:
done